app/index.js

```diff
@@ -99,11 +98,9 @@ module.exports = yeoman.generators.Base.extend({
 		this.prompt(prompts, function (props) {
 			var componentName = props.componentName;
 
-			var capitalizeName = _.startCase(componentName);
-
 			this.camelCaseName = _.camelCase(componentName);
-			this.capCaseName = capitalizeName.replace(' ', '');
-			this.capitalizeName = capitalizeName;
+			this.capCaseName = componentName;
+			this.capitalizeName = _.startCase(componentName);
```

Comment:
I thought that this one was the version of the name with spaces, since it was used before the `capitalizeName.replace(' ', '')` call before you've changed the input to not have spaces right? Now it seems like it's just the same as `capCaseName`, since `startCase` wouldn't do anything now that the string already starts with capital letter right? When would you use this instead of the other then?